### ISSUE TITLE
Unable to set SORT_PROPERTIES_ALPHABETICALLY in Jackson's config

### ISSUE THREAD

**Issue by lehphyro** [diffplug/spotless#2044]
It'd be nice to format json/yaml files with alphabetically sorted properties for consistency reasons, unfortunately Jackson's `SORT_PROPERTIES_ALPHABETICALLY` is a `MapperFeature` so it cannot be enabled through Spotless right now because it only accepts `SerializationFeature`.

**Comment by blacelle** [comment]
Jackson-based linters accepts both `SerializationFeature ` and `JsonGenerator.Feature| YAMLGenerator.Feature `.

I overlooked MapperFeature while introducing Jackson linters, though it should be fairly manageable to add them.

You may to rely on something like `JsonMapper jsonMapper = JsonMapper.builder(jsonFactory).enable(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY).build();`

Around: 
- https://github.com/diffplug/spotless/blob/main/lib/src/main/java/com/diffplug/spotless/json/JacksonConfig.java
- https://github.com/diffplug/spotless/blob/main/lib/src/jackson/java/com/diffplug/spotless/glue/json/AJacksonFormatterFunc.java#L75

**Comment by blacelle** [comment]
@Riduidel Yep, I provided guidelines about such a change in Spotless in my previous comment.
